plugins/modules/network/routeros/routeros_api.py

```diff
@@ -0,0 +1,494 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, Nikolay Dachev <nikolay@dachev.info>
+# GNU General Public License v3.0+ https://www.gnu.org/licenses/gpl-3.0.txt
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.0.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: routeros_api
+version_added: 1.0.1
```

Comment:
Don't put random values in there, it must be the value of the next release of the collection. Right now that's 1.0.0, but since that will be released today (and this PR won't get in by then), you can set it to the next one:
```suggestion
version_added: 1.1.0
```
(See #81 for information on releases. It could be that we do a 1.1.0 release if a new feature has been merged in a couple of weeks already; if we do that, probably on August 18th because that's the Ansible 2.10 collection freeze. If the PR misses this date, it eventually will make it into Ansible 2.10, but not into 2.10.0.)